src/components/SaveTimeRecording.js

```diff
@@ -38,26 +40,62 @@ class SaveTimeRecording extends Component {
 		});
 	}
 
+	handleCustomerChange(option) {
+		this.setState({ customer: option });
+	}
```

Comment:
Can't see a component test for this method being called

Reply:
added